schemas/cloudflare/terraform-resource-1.yml

```diff
@@ -156,8 +156,6 @@ properties:
     type: string
   kind:
     type: string
-  name:
-    type: string
```

Comment:
why is that removed?

Reply:
https://github.com/app-sre/qontract-schemas/pull/523/commits/dfa00c23074eb65c5fdf1450059beac9596a4b30

Reply:
Ah I see, duplicate...